codespell_lib/tests/test_basic.py

```diff
@@ -270,7 +270,16 @@ def test_ignore(tmpdir):
 
 def test_check_filename(tmpdir):
     """Test filename check."""
+    # Irregular file (!isfile())
+    fifodir = str(tmpdir)
+    os.mkfifo(op.join(fifodir, 'abandonned'))
+    assert cs.main('-f', fifodir) == 1
```

Comment:
mkfifo is only on Unix, do we run on Windows? I guess we need to skip this test on those platforms somehow?